members/SNJU31582_631485.yaml

```diff
@@ -0,0 +1,57 @@
+bioguide: SNJU31582_631485
+contact_form:
+  driver: chrome
+  steps:
+  - visit: "https://www.njleg.state.nj.us/legislative-roster/439/senator-durr"
```

Comment:
this needs to be https://www.njleg.state.nj.us/legislative-roster/439/senator-durr/contact

Reply:
fixed